docs/ai_actions/ai_actions_guide.md

```diff
@@ -0,0 +1,143 @@
+
+Out-of-the-box, AI Actions includes two essential components: a framework package and an OpenAI connector package.
+It comes pre-configured with the following action types:
+
+- [Refine text](#refining-text): Rewrite existing text according to instructions set in a prompt
+- [Generate alternative text](#generating-alternative-text): Generate alt text for images for accessibility purposes
+
+![AI Actions schematic](img/guide_ai_actions.png)
+
+You can extend its capabilities beyond the default setup by creating custom connector modules, allowing users to integrate additional AI services or customize the way data is processed and interpreted.
+For example, it could transform images, or generate illustrations for your articles based on their contents.
+The possibilities are endless and you're not limited to a specific AI service, avoiding vendor lock-in.
+
+## Availability
+
+AI Actions are an opt-in capability available as an LTS Update to the v4.6.x version of [[= product_name =]], regardless of its edition.
+To begin using AI Actions, you must first [install the required packages and perform initial configuration](install_ai_actions.md).
+
+!!! note "API Key"
+
+    The OpenAI connector requires that you first [get an API key](https://help.openai.com/en/articles/4936850-where-do-i-find-my-openai-api-key) and make sure that you [set up a billing method](https://help.openai.com/en/articles/9038407-how-can-i-set-up-billing-for-my-account).
+
+## How it works
+
+AI Actions LTS update relies on an extensible AI framework written in PHP.
+The framework package is responsible for gathering information from various sources, such as AI action types, AI action configurations, and contextual details like SiteAccess, user details, locale settings, and more.
+This data can then be combined with user input.
+It's then passed to a service connector, such as the default OpenAI connector, for final processing on [[= product_name =]] side.
+The service connector wraps all data into a prompt or another suitable format and sends it to an external AI service.
+
+When the AI Service returns a response, the response goes back through the service connector and passes to the framework.
+It can then be presented to the user in any way necessary.
+
+### Core concepts
+
+#### AI service
+
+AI service is a third party platform that provides access to artificial intelligence tools and capabilities.
+It executes tasks that it receives through a service connector.
+
+#### Action
+
+Actions are tasks or functions that the AI service is set to perform.
+Each action is a combination of an AI action type and an AI action configuration.
+Action types define what kind of task the AI service performs, while AI action configurations specify how the task should be executed.
+This clear separation allows for a flexible system where actions can be created, managed, and customized with minimal effort.
+
+#### AI action type
+
+AI action types are templates predefined by developers.
```

Comment:
It is crucial to define Action types as being high level. They match _users tasks_. Image to text isn't an action type, it's a handler (sorta). We need to give examples such as Generate the alternative text of an image.

Reason: the UI component is set to one particular action type, and that it will list _all_ the actions configured for that action type.